docs/tech/https.md

```diff
@@ -0,0 +1,118 @@
+# HTTPS
+
+[HTTPS](https://en.wikipedia.org/wiki/HTTPS) secures information when it's
+being transmitted over the internet. Everybody should use HTTPS, but we especially
+must use HTTPS because of the sensitive data we transmit.
```

Comment:
"we especially must" is a bit awkward... maybe "but it is especially important that we use HTTPS because..."